utils/healthcheck.py

```diff
@@ -75,7 +75,7 @@ def check_working_directory():
         check_working_directory.remediation = f'Please start LSDC in {daq_utils.beamline} data directory. Current directory: {working_dir}'
         return False
     if daq_utils.getBlConfig("visitDirectory") != os.getcwd():
-        check_working_directory.remediation = ("Working directory mismatch. Please start LSDC GUI in the same folder as the server is running.")
```

Comment:
This line was constructed in the original commit in such a way that the users could not have the possibility of seeing any metadata that might be encoded in the other user's directory name.

Reply:
I think this needs to be addressed.

Reply:
reverted with [0ebc395](https://github.com/NSLS-II/lsdc/pull/315/commits/0ebc3954d2a355a2a284b3e3b4e6345b6286b981)